### ISSUE TITLE
[Workspaces] Reset button is not accessible

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#28217]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1. Open Edit workspaces window
2. Try to navigate Reset button

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/3fcecd2e-8099-40f2-87b4-55176e82a841

### What is the latest version of MuseScore Studio where this issue is present?

4.6 nightly

### Regression

No.

### Operating system

all

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
